syntax_body/java/hilink/javadoc.vim

```diff
@@ -0,0 +1,8 @@
+hi def link javaDocTags     Special
+hi def link javaDocParam        Function
+hi def link javaDocSeeTagParam      Function
+hi def link javaDocComment      Comment
+hi def link javaCommentTitle		SpecialComment
+
+hi def link htmlComment		Special
+hi def link htmlCommentPart		Special
```

Comment:
I don't like that spaces and tabs are used together.
So I would like you to use only spaces like other files on this repository.

Reply:
Ok! I have format it as other files in this repository. Please have a look and point out is there other things that I need to do? Thank you.